src/olympia/amo/tests/__init__.py

```diff
@@ -678,6 +682,14 @@ def addon_factory(
     for user in users:
         addon.addonuser_set.create(user=user)
 
+    application = version_kw.get('application', amo.FIREFOX.id)
+    if not category:
+        category = Category.from_static_category(
+            next(CATEGORIES[application][type_].itervalues()))
```

Comment:
Mmmm, this does not ensure that the category does not exist yet, maybe it should (through `.get_or_create` with the right `defaults` instead of `from_static_category` maybe ?)